deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml

```diff
@@ -1375,7 +1382,8 @@ spec:
           required:
           - controller
           - node
-          type: object
+          - license
```

Comment:
not sure what this is, but any changes in crd should also be done in the generated yaml, and in the latest CSV yamls

Reply:
can you specify the relevant files?

Reply:
https://github.com/IBM/ibm-block-csi-operator/search?q=ibmblockcsis.csi.ibm.com
(excluding paths with a previous version number)

Reply:
updated

Reply:
**excluding paths with a previous version number**.
as I [mentioned before](https://github.com/IBM/ibm-block-csi-operator/pull/106#discussion_r507141902), we are not allowed to change CSV yamls of previous versions.

Reply:
@oriyarde , pls check

Reply:
@barakda please go over the diff and make sure there are no changes to [CSV yamls of previous versions](https://github.com/IBM/ibm-block-csi-operator/pull/112/files#diff-391ed340aea63b1b88ac863444be8c9b186ef4dcf17901083fb50c7b9486730c)

Reply:
@oriyarde , have you found another place that the license needs to be added ?

Reply:
no, just extra change in non-1.4.0